src/main/kotlin/com/approvers/devlazaApi/controller/ExtendsFunctions.kt

```diff
@@ -0,0 +1,85 @@
+package com.approvers.devlazaApi.controller
```

Comment:
こいつらはControllerではないのでcontroller以下にあるのはまずい。

```suggestion
package com.approvers.devlazaApi.util
```